Source/Client/Sync/SyncHandlers.cs

```diff
@@ -302,7 +302,7 @@ static void RenameZone(Dialog_RenameZone __instance)
         [MpPrefix(typeof(TransferableUIUtility), "DoCountAdjustInterface")]
         static void TransferableAdjustTo(Transferable trad)
         {
-            var session = MpTradeSession.current ?? (ISessionWithTransferables)MpFormingCaravanWindow.drawing?.Session ?? MpLoadTransportersWindow.drawing?.Session;
+            var session = MpTradeSession.current ?? Multiplayer.WorldComp.splitSession ?? (ISessionWithTransferables)MpFormingCaravanWindow.drawing?.Session ?? MpLoadTransportersWindow.drawing?.Session;
```

Comment:
This line needs to be wrapped.

Consider prefixing this with a null check for `MpFormingCaravanWindow.drawing`, since you are using conditional access twice in the same line for this field.

Reply:
I'll wrap the line. Not touching MpFormingCaravanWindow.drawing it's outside the scope of my change. Also it fine as-is.